draft/schemas/context-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "https://raw.githubusercontent.com/dini-ag-kim/lrmi-profile/23-modularize/draft/schemas/context-schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "JSON-LD Context",
+    "description": "The JSON-LD context for the structured resource descriptions",
+    "type": "array",
+    "minItems": 2,
+    "items": [
+      {
+        "type": "string",
+        "format": "uri",
+        "enum": [
+          "https://w3id.org/kim/lrmi-profile/draft/context.jsonld"
+        ]
+      },
+      {
+        "type": "object",
+        "properties": {
+          "@language": {
+            "$ref": "https://raw.githubusercontent.com/dini-ag-kim/lrmi-profile/23-modularize/draft/schemas/Language.json"
```

Comment:
This does not seem to work as expected.
When trying to publish something, it throws an error because we have `"minItems": 2`, but the field for language selection for `"@context"` does not show up in the editor.
I recommend setting `"minItems": 1`  and `required` to an empty array for now till this is fixed